scss/_login_reg.scss

```diff
@@ -0,0 +1,51 @@
+
+/*input*/
+input[type=text],input[type=password], select  {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 5px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+
+}
+
+
+.btn {
+  width: 30%;
+  background-color: #4CAF50;
```

Comment:
Farben als variable in colors.scss hinterlegen; nie als direkten wert hinzufügen

Reply:
Mehrmals in dieser Datei, bitte selber schauen